### PR TITLE
Goreleaser brew

### PR DESCRIPTION
**Pull request by StanGirard**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,6 @@ dist/
 
 # Test artifacts
 internal/e2e/testdata/**/moves.tf
+
+# Envrc
+.envrc
```
Comment (busser) [inline .gitignore:16]: I agree that this should be added to .gitignore 👍

However I think we can make the comment clearer:
```suggestion
# Local direnv configuration
.envrc
```

**Review by busser — CHANGES_REQUESTED** [review]
How can we test that this works well?

Also, should we add the tfautomv-maintainers team as maintainers to the new tap repository?

**Comment by StanGirard** [comment]
@busser The issue is we can't. Just try it. If it fails, delete the last commits and release and retry. 

You need to have the github token in your env variables that is all.

**Comment by StanGirard** [comment]
And yes tfautomv-maintainers should be added as maintainers or we could let anybody at Padok use it.

I'd go for the second option.

**Pull request merged**